docs/version_control.md

```diff
@@ -44,3 +63,21 @@ We recommend a simple flow based on following rules:
 ### Comment styles
 
 We encourage using [Commitizen](http://commitizen.github.io/cz-cli/), a great tool for writing angular commits - this will create a standardized commit format which makes for easier change logging and more sane messages.
+  
+### Alternative options 
```

Comment:
I think this content is best before the ## Best Procatices section

Maybe in best practices we have an explicit explanation about workflow

I'll put a suggestion below